### ISSUE TITLE
Buy players !!

### ISSUE THREAD

**Issue by Amrofifa** [galodoido794/FutAUTO#7]
When we can buy players ?? 
Thx

**Comment by galodoido794** [comment]
It has always been free and I get nothing from it. So I am thinking of not spending more time on this project or ending the project.

**Comment by BadSiberian** [comment]
no, please keep working on the project. can add paid functionality to the application or make a paid full version.
